public/docs/ts/latest/tutorial/_data.json

```diff
@@ -31,8 +31,8 @@
     "nextable": true
   },
   "toh-pt6": {
-    "title": "Http",
-    "intro": "We convert our service and components to use Http",
+    "title": "HTTP",
```

Comment:
LOL. I just REVERSED that.  The class is `Http`, not `HTTP`.   The module is `HttpModule` not `HTTPModule`.